packages/common/src/Utils.ts

```diff
@@ -314,3 +329,24 @@ export function formatTokenAmount (balance: BN, tokenDecimals: BN, tokenSymbol:
   }
   return `${fromWei(shiftedBalance)} ${tokenSymbol}`
 }
+
+export function splitRelayUrlForRegistrar (url: string, partsCount: number = 3): string[] {
+  const maxLength = 32 * partsCount
+  if (url.length > maxLength) {
+    throw new Error(`The URL does not fit to the RelayRegistrar. Please shorten it to less than ${maxLength} characters`)
+  }
+  const parts = url.match(/.{1,32}/g) ?? []
```

Comment:
I think this does the entire split:
```
Buffer.from(url).toString('hex').match(/.{2,64}/g).map(x=>'0x'+x.padEnd(64,'0'))
```

Reply:
This does not fill in the array of 3, therefore it is not much better.

Reply:
Buffer.from(url).toString('hex').padEnd(96*2,'0') .